pkgs/top-level/all-packages.nix

```diff
@@ -18777,6 +18776,18 @@ with pkgs;
 
   folly = callPackage ../development/libraries/folly { };
 
+  fbthrift = callPackage ../development/libraries/fbthrift { };
+
+  fb303 = callPackage ../development/libraries/fb303 { };
+
+  edencommon = callPackage ../development/libraries/edencommon { };
+
+  fizz = callPackage ../development/libraries/fizz { };
```

Comment:
We don't really sort all-packages that much but libraries should be near libraries and not applications.

Reply:
Hey @SuperSandro2000, maybe I'm blind but all these libraries appear to be near other libraries.